docs/src/2.3.RenderingObjects.mdx

```diff
@@ -0,0 +1,196 @@
+import CodeSandboxEmbed from './jsx/utils/CodeSandboxEmbed.js';
+
+# Rendering Objects
+
+In this page, you'll learn how to:
+
+- build a Hello World app with regl-worldview
+- render objects with custom position, color and scale
+- apply math equation to the rendered objects
+- use instanced rendering
+
+## Building a Hello World App
+
+Add regl-worldview to the package dependencies:
+
+```bash
+"regl-worldview": "latest"
+```
+
+Create a new file `Example.js` in the `src/` directory, add below code which renders a Worldview component with a red cube and x/y/z axes inside:
+
+```js
+// src/Example.js
+import React from "react";
+import Worldview, { Cubes, Axes } from "regl-worldview";
+
+export default function Example() {
+  return (
+    <Worldview>
+      <Cubes>
+        {[
+          {
+            pose: {
+              orientation: { x: 0, y: 0, z: 0, w: 1 },
+              // position it at the center
+              position: { x: 0, y: 0, z: 0 },
+            },
+            scale: { x: 10, y: 10, z: 10 },
+            // red color. The rgba value of the color is between 0 and 1 (inclusive).
+            // http://wiki.ros.org/rviz/DisplayTypes/Marker
+            color: { r: 1, g: 0, b: 0, a: 1 },
+          },
+        ]}
+      </Cubes>
+      <Axes />
+    </Worldview>
+  );
+}
+```
+
+Replace the contents of `src/index.js` with the following code:
+
+```js
+// src/index.js
+import React from "react";
+import ReactDOM from "react-dom";
+import Example from "./Example";
+
+function App() {
+  return (
+    <div style={{ width: "100vw", height: "100vh" }}>
+      <Example />
+    </div>
+  );
+}
+
+const rootElement = document.getElementById("root");
+ReactDOM.render(<App />, rootElement);
+```
+
+Now you have successfully created a HelloWorld 3D app! In the browser preview, you'll see the red cube and axes. You can click and interact with Worldview with one finger (move the scene) or two fingers (rotate the scene). Learn more about [controlling the camera with keyboard](#/docs/api/worldview).
```

Comment:
> one finger (move the scene) or two fingers (rotate the scene)

I think that's incorrect as we dont have touch interactions yet :(, it should be `left click` and `right click`.

'You can interact with Worldview with the left click to move the scene, or right click to rotate the scene'